src/main/resources/db/migration/V1__create_users_table.sql

```diff
@@ -0,0 +1,8 @@
+# V1__create_users_table.sql
+
+CREATE TABLE my_user (
+                         id BIGINT AUTO_INCREMENT PRIMARY KEY,
+                         login_name VARCHAR(255) NOT NULL,
+                         password VARCHAR(255) NOT NULL,
+                         role ENUM('USER', 'ADMIN' , 'SUPER_ADMIN') NOT NULL DEFAULT 'USER'
+);
```

Comment:
missing empty line at end of this file

Reply:
je dobry zvyk nechavat jeden prazdny riadok na konci aj github tam potom hadze taky cerveny znak ale nemusis to riesit ak sa ti nechce

Reply:
opravene